models/base_gan_model.py

```diff
@@ -479,12 +474,40 @@ def compute_G_loss_GAN(self):
                 loss_name,
                 loss_value,
             )
-
             self.loss_G_tot += loss_value
 
         if self.opt.train_temporal_criterion:
             self.compute_temporal_criterion_loss()
 
+    def compute_fake_with_context(self, fake_name, real_name):
+        setattr(
+            self,
+            fake_name + "_with_context",
+            torch.nn.functional.pad(
+                getattr(self, fake_name),
+                (
+                    self.opt.data_online_context_pixels,
+                    self.opt.data_online_context_pixels,
+                    self.opt.data_online_context_pixels,
+                    self.opt.data_online_context_pixels,
+                ),
+            ),
+        )
+
+        setattr(
+            self,
+            fake_name + "_with_context",
+            getattr(self, fake_name + "_with_context")
+            + self.mask_context * getattr(self, real_name + "_with_context"),
+        )
+        setattr(
+            self,
+            fake_name + "_with_context_vis",
+            torch.nn.functional.interpolate(
+                getattr(self, fake_name + "_with_context"), size=self.real_A.shape[2:]
+            ),
+        )
+
```

Comment:
All changes above are not from this PR, I believe they shouldn't be here, right ?

Reply:
Those codes were not written by me, and I have no clue where they came from. I had an accidental pull three weeks ago, which was fixed by Louis; I'm wondering if they were imported at that time. Can I delete them without destroying the work of others?"

Reply:
they are being removed